pyproject.toml

```
@@ -26,7 +26,7 @@ classifiers = [
 ]
 dependencies = [
     "Sphinx>=4.2.0",
-    "pydata-sphinx-theme>=0.14.0",
+    "pydata-sphinx-theme==0.14.4",
```

```suggestion
    "pydata-sphinx-theme>=0.14,<0.15",
```

Something like this seems better suited